roles/softserve/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Install required packages
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - python-virtualenv
+  - gcc
+  - git
+  - python-devel
+  - postgresql-devel
+
+- name: Add tmpfiles snippet
+  template:
+    dest: /etc/tmpfiles.d/softserve.conf
+    src: softserve.tmpfiles.conf
+  notify: Create systemd tmpfile
+
+- name: Add gunicorn systemd service
+  template:
+    dest: /etc/systemd/system/softserve.service
+    src: softserve.service
+  notify: Reload systemd
+
+- name: Start the softserve service
```

Comment:
You need to flush the handlers if you want systemd to see the service, see 'meta: flush_handlers" somewhere in the existing repo of playbook for example. Otherwise, initial deploy will fail.

Reply:
It would also be better to start the service at the end, when the code is deployed, or it will fail too.